steampy/client.py

```diff
@@ -311,6 +313,17 @@ def fetch_price(self, item_hash_name: str, game: GameOptions, currency: str = Cu
             raise TooManyRequests("You can fetch maximum 20 prices in 60s period")
         return response.json()
 
+    @login_required
+    def get_my_market_listings(self) -> dict:
+        response = self._session.get("%s/market" % SteamClient.COMMUNITY_URL)
+        if response.status_code == 200:
+            assets_descriptions = json.loads(text_between(response.text, "var g_rgAssets = ", ";"))
```

Comment:
We are getting a dict which contains the usual descriptions.
Example of the structure [here](https://gist.github.com/Ermans/087a8cf4413bc6e7cb6330a4064a5587)